src/components/pages/faq/introduction.tsx

```diff
@@ -1,9 +1,10 @@
+/* eslint-disable no-irregular-whitespace */
```

Comment:
Why was this needed?

Reply:
Something weird was happening with copy paste and I forgot to remove it. Fixed.